tensorflow/python/kernel_tests/variable_scope_test.py

```diff
@@ -94,6 +94,23 @@ def testInitFromNonTensorValue(self):
       with self.assertRaises(TypeError):
         tf.get_variable("x", initializer={})
 
+  def testInitFromNonInitializer(self):
+    with self.test_session() as sess:
+      # Test various dtypes with zeros initializer as following:
+      TYPES = [tf.int8, tf.uint8, tf.int16, tf.uint16, tf.int32, tf.int64, 
```

Comment:
capital letters only allowed for module-level static constants.  this should be just 'types'.